### DISCUSSION TITLE
Deployement issue - Longin impossible with filament

### DISCUSSION THREAD

**Discussion by scoba6** [filamentphp/filament#6446] [General]
Hello everybody,

I'm facing on a big problem after the deployment of my app online with namecheap. Impossible to log on the app with error 500 in filament.core.auth.login as you can on the screen. I'm looking for solutions lot of day ago but nothing works. I think it's probably a bug from livewire but don't know exactly. Please help as possible. @danharrin @zepfietje 
<img width="1007" alt="500" src="https://user-images.githubusercontent.com/30085964/236692158-82af3d44-98f2-456c-93a1-33b4a44caff2.png">

**Comment by danharrin** [comment]
Your server is misconfigured. Do not host Laravel apps in a subdirectory of your domain, its bad practice. Put it on the root of the domain or on a subdomain. That will probably fix your issues.

Also I wouldnt reccomend shared hosting. Use Laravel Forge or Vapor and you wont face any challenges.

**Reply by scoba6** [reply]
Tnanks for your help, I try to apply your advise and feed back when it works.
